lib/prmd/commands/combine.rb

```diff
@@ -79,7 +79,13 @@ def self.combine(paths, options = {})
       base = Prmd::Template.load_json('combine_head.json')
       schema = base['$schema']
       meta = {}
-      meta = Prmd.load_schema_file(options[:meta]) if options[:meta]
+      if filename = options[:meta]
+        meta = Prmd.load_schema_file(filename)
+        if meta.nil?
```

Comment:
A `present?` from active support would be preffered, since nil doesn't respond to `#empty?`, so instead I"ll combine both into a `meta.nil? || meta.empty?`